activity_browser/bwutils/ecoinvent_biosphere_versions/ecospold2biosphereimporter.py

```diff
@@ -15,15 +15,18 @@
 logger = logging.getLogger('ab_logs')
 log = ABHandler.setup_with_logger(logger, __name__)
 
+
 def create_default_biosphere3(version) -> None:
     """Reimplementation of bw.create_default_biosphere3 to allow import from older biosphere versions."""
     # format version number to only Major/Minor
     version = version[:3]
 
-    if version == sort_semantic_versions(__ei_versions__)[0]:
+    if version == sort_semantic_versions(__ei_versions__)[0][:3]:
+        log.debug(f'Installing biosphere version >{version}<')
```

Comment:
Not sure if just me, but I don't see this log showing up anywhere. Not in the console nor in the .log files...

Probably not an issue with your code though.

Reply:
#1239